demos/CMakeLists.txt

```diff
@@ -1,33 +1,103 @@
-project (nbody)
+cmake_policy(SET CMP0091 NEW)
```

Comment:
Can you please add a short comment what this does and why do we need it?

Reply:
Done